src/Kdyby/Doctrine/Entities/BaseEntity.php

```diff
@@ -25,6 +25,7 @@
 /**
  * @author Filip Procházka <filip@prochazka.su>
  *
+ * @deprecated
  * @ORM\MappedSuperclass()
  */
 abstract class BaseEntity extends Nette\Object implements \Serializable
```

Comment:
Up untill recently (which wasn't intentional anyway) there was nothing that would prevent you from using it with 5.3. I saw no added value in just breaking the compatibility for the sake of breaking it. So I just removed the build from travis and let it live :)